### DISCUSSION TITLE
How to add TailwindCSS?

### DISCUSSION THREAD

**Discussion by Worthy-Programmer** [crewdevio/Snel#26] [Q&A] [answered]
Hey sorry very much, this is not an issue but an act of ignorance. How to add Tailwind CSS in snel?

**Comment by buttercubz** [comment, marked as answer]
hi, yes tailwind can be used with snel, I am currently creating an app with snel and tailwind, the only downside is that it must be used by cdn and it's a bit limited that way.

in your index.html

```html
<link href="https://unpkg.com/tailwindcss/dist/tailwind.min.css" rel="stylesheet"/>
```

**Comment by Worthy-Programmer** [comment]
Hey thank you, is there any other way? Like adding to postcss of rollup. Idk Im not well into rollup.

**Comment by buttercubz** [comment]
> Hey thank you, is there any other way? Like adding to postcss of rollup. Idk Im not well into rollup.

snel have already include postcss support

https://github.com/crewdevio/Snel/blob/1e16fff70848d910e2364bbe42d9f5f7f61465d6/compiler/build.ts#L40

**Comment by aadamsx** [comment]
Do you have an example app with tailwind working with postcss and rollup?

**Reply by Worthy-Programmer** [reply]
No I don't. I'm not that good in postcss and rollup. So I switched the project to Svelte.

**Comment by aadamsx** [comment]
I'm really interested in this project, but need/want to use tailwind for css -- has anyone got this working?

**Reply by buttercubz** [reply]
hi @aadamsx Currently you can use tailwind with snel using a cdn, the bad thing is that some features cannot be used, since tailwind that depends a lot on the nodejs ecosystem and for now I don't think they have plans to support deno
